pkg/cnab/extensions/dependencies.go

```diff
@@ -94,7 +101,27 @@ func HasDependencies(bun bundle.Bundle) bool {
 	return ok
 }
 
+
+// ListBySequence returns the dependencies by the defined sequence,
+// if none is specified, they are unsorted.
+func (d Dependencies) ListBySequence() []Dependency {
```

Comment:
Why not return a `Dependencies` object as opposed to `[]Dependency`?

Reply:
This function is responsible for understanding how to convert the data inside of a Dependencies (sequence and requires) and return a sequence of Dependency items for the consumer to iterate over. This way the logic from combining sequence and requires is in a single location.

It wouldn't really help to return a Dependencies struct from this method.